.github/workflows/drupalCodingStandards.yml

```diff
@@ -0,0 +1,19 @@
+name: Drupal coding standards
+
+on: [pull_request]
+
+jobs:
+  phpcs:
+    name: PHPCS
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Install composer depdencies so we have drupal/coder which adds our codesniffer, the bin/phpcs file
+      # and dealerdirect/phpcodesniffer-composer-installer which changes the installer_path configuring all sniffs.
+      - name: Install dependencies
+        run: composer install --dev --prefer-dist --no-progress --no-suggest
```

Comment:
Do we install all composer dependencies here? Or just Open Social?

Reply:
its running in the root, so it's adding all from the distributions composer.json including the dev. (There is no such a thing as just installing dev without the normal unfortunately)

Reply:
Check, ok!